openapi/components/schemas/GatewayAccountConfig/PayPal.yaml

```
@@ -5,6 +5,23 @@ allOf:
     required:
       - settings
     properties:
+      credentials:
+        type: object
+        description: PayPal credentials object. The credentials object may be provided in order to specify the PayPal Partner account under which the gateway account is onboarded.
+        properties:
+          partnerId:
+            type: string
+            description: ID of the PayPal Partner account.
+          partnerClientId:
+            type: string
+            description: ID of the PayPal Partner account API client.
+          partnerSecret:
+            type: string
+            description: Secret key for the PayPal Partner account API client.
+            format: password
+          partnerBnCode:
+            type: string
+            description: BN Code for the PayPal Partner account. Used to track the source of API requests.
```

nitpick: suggestion to avoid passive voice
```suggestion
            description: BN Code for the PayPal Partner account. This unique code tracks the source of API requests.
```